forge-gui/res/cardsfolder/upcoming/kardum_patron_of_flames.txt

```diff
@@ -6,10 +6,14 @@ K:Haste
 T:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME attacks, put a flame counter on it, then seek a card with mana value equal to the number of flame counters on it and exile that card face down.
 SVar:TrigCounter:DB$ PutCounter | CounterType$ FLAME | CounterNum$ 1 | ValidCards$ Self | SubAbility$ DBSeek
 SVar:DBSeek:DB$ ChangeZone | Origin$ Library | Destination$ Exile | ExileFaceDown$ True | AtRandom$ True | NoShuffle$ True | NoLooking$ True | NoReveal$ True | ChangeNum$ 1 | ChangeType$ Card.cmcEQX+YouCtrl | RememberChanged$ True 
-T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigReturn | ForgetOnMoved$ Exile | TriggerDescription$ When CARDNAME dies, put the cards exiled with it into their owner's hand.
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard | Execute$ TrigReturn | TriggerDescription$ When CARDNAME dies, put the cards exiled with it into their owner's hand.
 SVar:TrigReturn:DB$ ChangeZoneAll | ChangeType$ Card.IsRemembered+ExiledWithSource | RememberChanged$ True | Origin$ Exile | Destination$ Hand | SubAbility$ DBDelay
 SVar:DBDelay:DB$ DelayedTrigger | DelayedTriggerDefinedPlayer$ You | Mode$ Phase | Phase$ End of Turn | Execute$ TrigDiscardExiled | RememberObjects$ Remembered | TriggerDescription$ At the beginning of the end step of your next turn, discard those cards.
 SVar:TrigDiscardExiled:DB$ Discard | Mode$ Defined | DelayedTriggerDefinedPlayer$ You | DefinedCards$ DelayTriggerRemembered
+T:Mode$ ChangesZone | Origin$ Battlefield | Destination$ Exile | Static$ True | ValidCard$ Card.Self | Execute$ DBForget
```

Comment:
OOH look at `Helvault`: `ExcludedDestinations$ Graveyard | Destination$ Any` very cool

You might update Toluz to make it cleaner, too. Feel free to add that to this PR since it's related 😉